src/account/accountsSelectors.js

```diff
@@ -14,10 +14,10 @@ export type AccountStatus = {| ...Identity, isLoggedIn: boolean |};
  * This should be used in preference to `getAccounts` where we don't
  * actually need the API keys, but just need to know whether we have them.
  */
-export const getAccountStatuses: Selector<$ReadOnlyArray<AccountStatus>> = createSelector(
-  getAccounts,
-  accounts =>
-    accounts.map(({ realm, email, apiKey }) => ({ realm, email, isLoggedIn: apiKey !== '' })),
+export const getAccountStatuses: Selector<
+  $ReadOnlyArray<AccountStatus>,
+> = createSelector(getAccounts, accounts =>
+  accounts.map(({ realm, email, apiKey }) => ({ realm, email, isLoggedIn: apiKey !== '' })),
```

Comment:
And this change seems one step forward, one step back -- the more compact `createSelector` arguments seem nice, but the type parameter set off on its own line is silly. On net, probably not worth fighting Prettier over.